CMakeLists.txt

```diff
@@ -37,11 +36,10 @@ option(BUILD_SHARED_LIBS "Set ON to build a shared library" OFF)
 set (IO_BACKEND "mpi" CACHE STRING "Default IO backend (mpi (default) or adios2)")
 set_property(CACHE IO_BACKEND PROPERTY STRINGS mpi adios2)
 
-set(AUTHOR "Stefano Rolfo;Charles Moulinec;Paul Bartholomew")
+set(AUTHOR "The 2DECOMP&FFT team")
 set(AUTHOR_DETAILS "stefano.rolfo@stfc.ac.uk;charles.moulinec@stfc.ac.uk;p.bartholomew@epcc.ed.ac.uk")
```

Comment:
Should we change this line to point to the CONTRIBUTORS file or similar?

Reply:
I hald modified this. I want to remove the email and just point to the dev team